### DISCUSSION TITLE
Can renovate create yarn resolutions?

### DISCUSSION THREAD

**Discussion by zjullion** [renovatebot/renovate#15574] [Request Help] [answered]
Hi,

Consider a simple repo with the following in `package.json`:

```
{
  // other package.json keys
  "dependencies": {
    "storybook-readme": "5.0.9"
  }
}
```

There are a number of security vulnerabilities present in `storybook-readme@5.0.9`, and no newer version is available. Most of these vulnerabilities can be fixed with `resolutions`, as so:

```
  "resolutions": {
    "highlight.js": "^10.4.1",
    "prismjs": "^1.25.0",
  }
```

Is there a configuration option to get `renovate` to open PRs with these `resolutions`? Is the GitHub App capable of this?

**Comment by viceice** [comment, marked as answer]
no, and i don't think we will implement such a feature. sorry

**Reply by zjullion** [reply]
May I ask why not? It's very useful for fixing security issues.

**Reply by rarkins** [reply]
If you can propose a PR with an elegant way to achieve this, we'll consider it
